src/components/common/EthHashInfo/SrcEthHashInfo/index.tsx

```diff
@@ -35,8 +35,8 @@ const SrcEthHashInfo = ({
   address,
   customAvatar,
   prefix = '',
-  copyPrefix,
-  showPrefix,
+  copyPrefix = true,
+  showPrefix = true,
```

Comment:
Why not remove this entirely?

Reply:
I thought there might be some use cases where you want to display just the hash, maybe the prefix is rendered externally or not at all necessary (e.g. a transaction hash).